editor/connections_dialog.cpp

```diff
@@ -1008,6 +1008,8 @@ void ConnectionsDock::_connect(const ConnectDialog::ConnectionData &p_cd) {
 	undo_redo->add_undo_method(SceneTreeDock::get_singleton()->get_tree_editor(), "update_tree");
 
 	undo_redo->commit_action();
+
+	ScriptEditor::get_singleton()->update_connected_methods();
```

Comment:
```suggestion
	undo_redo->add_do_method(ScriptEditor::get_singleton(), "update_connected_methods");
	undo_redo->add_undo_method(ScriptEditor::get_singleton(), "update_connected_methods");

	undo_redo->commit_action();
```

This should use the undo redo system so that it is also updated when undoing adding/removing connections.
Same with the other two calls.

`update_connected_methods` will also have to be bound for this, in `ScriptEditor::_bind_methods`:
`ClassDB::bind_method("update_connected_methods", &ScriptEditor::update_connected_methods);`